apps/somnia/project.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "somnia",
+  "$schema": "../../node_modules/nx/schemas/project-schema.json",
+  "sourceRoot": "apps/somnia/src",
+  "projectType": "application",
+  "tags": [],
+  "targets": {
+    "build": {
+      "executor": "@nx/esbuild:esbuild",
+      "outputs": ["{options.outputPath}"],
+      "defaultConfiguration": "production",
+      "options": {
+        "platform": "node",
+        "outputPath": "dist/apps/somnia",
+        "format": ["cjs"],
+        "bundle": true,
+        "main": "apps/somnia/src/main.ts",
+        "tsConfig": "apps/somnia/tsconfig.app.json",
+        "assets": [
+          "apps/somnia/src/assets",
+          "apps/somnia/src/data",
```

Comment:
Do we need all these paths here?

Reply:
i expect to use assets in the future for any somnia specific assets that arn't tied to other GO projects. data was meant for storing user data, can be used for temporarily keeping it before that backend database